en/docs/includes/db-advanced-config.md

```diff
@@ -11,6 +11,7 @@ Apart from the basic configurations specified above, WSO2 Identity Server suppor
 	validationQuery="SELECT 1"
 	validationInterval="30000"
 	defaultAutoCommit=false
+    commitOnReturn=true
```

Comment:
Can see a slight indentation issue.

Reply:
Fixed with https://github.com/wso2/docs-is/pull/3375/commits/ddfc78d45eacc188cbc0a192bec916591e7b78ae.